_posts/2013-01-11-navigate-using-just-your-keyboard.md

```diff
@@ -9,11 +9,26 @@ categories:
   - Quick Tests
 ---
 
-One quick way that you can check your accessibility is to navigate through the site using nothing but your keyboard.
+### Introduction
+Testing your site for keyboard navigation will ensure that keyboard users can access your site.
 
-1. Un-plug/turn off your mouse.
-2. Use the <kbd>Tab</kbd> key browse.
-3. Use <kbd>Enter</kbd> to select elements.
-4. Some menus and form elements may require <kbd>&uarr;</kbd> <kbd>&darr;</kbd> <kbd>&larr;</kbd> <kbd>&rarr;</kbd> arrow keys.
+As a bonus, sites that can be navigated with a keyboard can also likely be navigated using other input devices.
 
-This is by no means a complete accessibility check, but it can quickly help you find major accessibility problems.
+### How to navigate a website with a keyboard
+Generally, pressing `tab` moves focus between elements and pressing `enter` will interact with them.
+
+If you are using Safari, `option-tab` is the default keyboard shortcut to move focus between elements. You can update your settings to make `tab` the default navigation key by following the instructions in this article: [Quick Tip: Safari Keyboard Navigation](https://a11yproject.com/posts/safari-keyboard-navigation/)
+
+Additional keyboard shortcuts are described here: [WebAIM - Keyboard Accessibility](https://webaim.org/techniques/keyboard/)
+
+### Things to keep in mind while testing
+It is not enough to simply tab through a webpage from start to end. While navigating the site, keep these things in mind:
+- Interactive elements should be functional using only keyboard shortcuts.
```

Comment:
i don't think "shortcut" is the right word here.

maybe 
"Interactive elements should be functional using their default activation keys. e.g. <kbd>Enter</kbd> for links and <kbd>Space</kbd> for buttons."

as an example